apps/site/assets/ts/__v3api.d.ts

```diff
@@ -315,6 +324,10 @@ export interface Schedule {
   train_number?: string;
 }
 
+export interface ScheduleForStop extends Omit<Schedule, "time"> {
+  time: Date;
+}
+
```

Comment:
Based on our slack discussion, we're in agreement that splitting this out to a separate file is worthwhile as a separate tech debt ticket. I would still recommend renaming `ScheduleForStop` since the existing `Schedule` interface is also for a stop. Or alternatively, consider renaming the old `Schedule` interface if it has a specific use-case, whereas this format seems like it can be more easily repurposed.

Reply:
Okay. I will moved `ScheduleForStop` and `PredictionForStop` to their own files respectively.  Also rename them to `ScheduleWithTimestamp` and `PredictionForTimestamp`